tests/browsercontext-route.spec.ts

```diff
@@ -123,8 +123,9 @@ it('should support Set-Cookie header', async ({ contextFactory, server, browserN
     });
   });
   await page.goto('https://example.com');
+  const defaultSameSiteCookieValue = browserName === 'chromium' || (browserName === 'firefox' && browserMajorVersion >= 96) ? 'Lax' : 'None';
```

Comment:
Revert changes in the tests

Reply:
discussed offline and decided to land here

Reply:
reverted